apps/tenant-management-webapp/src/app/pages/admin/services/script/styled-components.ts

```diff
@@ -501,3 +501,19 @@ export const OuterNotificationTemplateEditorContainer = styled.div`
   overflow: hidden;
   margin-top: 90px;
 `;
+
```

Comment:
CustomLoaders repeat many times, could be extract into as component .

 {customIndicator && (
            <CustomLoader>
              <GoACircularProgress size="small" visible={true} />
            </CustomLoader>
          )}